test.sh

```diff
@@ -17,15 +11,15 @@ uv sync --all-extras
 source .venv/bin/activate
 
 # *** build ***
-uv run scons -j8
+uv run scons -j$(nproc 2>/dev/null || sysctl -n hw.logicalcpu)
 
 # *** lint ***
 # TODO: pre-commit is slow; replace it with openpilot's "op lint"
 #uv run pre-commit run --all-files
 uv run ruff check .
 
 # *** test ***
-uv run pytest -n8
+uv run pytest -n$(nproc 2>/dev/null || sysctl -n hw.logicalcpu)
```

Comment:
this was intentional. the tests actually run slower with more cores since pytest has tons of overhead on distributing the tests